app/plugins/router.js

```diff
@@ -1,14 +1,14 @@
 import health from '../routes/health.js'
 
-const plugin = {
+const router = {
```

Comment:
Name changed from `plugin` to `router` as SonarCloud points out that name of the default export and the file should match.